### ISSUE TITLE
FF-database data missing

### ISSUE THREAD

**Issue by itsnickchiu** [zhaoyuzhi/ChildPredictor#4]
Thanks for your excellent work and great effort for creating new kinship dataset!
FYI, I found that in the training dataset, data in folder 3823 are lost.

**Comment by zhaoyuzhi** [comment]
Yes, we will delete some image samples with poor quality in step (3). The full pipeline is as follows:

https://github.com/zhaoyuzhi/ChildPredictor/blob/main/img/data_collection.png

**Comment by itsnickchiu** [comment]
okay, got it. Thanks for your reply!
If that's the case, maybe you could consider to remove the following two files because their quality are also quite worse and besides they couldn't be detected using facial landmark to complete face alignment.

1. train/USA states/Hawaii family/2299/06222122.png
2. train/USA states/Utah family/3625/04221122.png

**Comment by zhaoyuzhi** [comment]
Thanks. Yes, these two faces are of low quality.
